src/System.Private.CoreLib/shared/System/Security/SecureString.Windows.cs

```diff
@@ -236,7 +237,7 @@ internal unsafe IntPtr MarshalToStringCore(bool globalAlloc, bool unicode)
         // -----------------------------
 
         private const int BlockSize = (int)Interop.Crypt32.CRYPTPROTECTMEMORY_BLOCK_SIZE / sizeof(char);
-        private SafeBSTRHandle _buffer;
+        private SafeBSTRHandle _buffer = null!;
```

Comment:
Shouldn't this actually be `SafeBSTRHandle?`?  It can actually be null after it's been disposed.

Reply:
I'm actually a bit torn on this - in theory yes - but I consider lifetime of the object from construction to dispose or finalizer (whichever comes first) and in that lifetime it cannot be null - after Dispose you're in the broken state. Do you think this should be nullable?